# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/ContentCompositorBridgeParent.cpp
# Commit: 7e2a2b1b416f
# Full Hash: 7e2a2b1b416f1a64c6548aab29e92acc9554ae7d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix all webgl regression tests according to CI. r=handyman
#   
#   (This is a combination of 31 commits)
#   
#   * Fix Linux compilation.
# ==============================================================================

diff -r ccfa767dba64 -r 7e2a2b1b416f gfx/layers/ipc/ContentCompositorBridgeParent.cpp
--- a/gfx/layers/ipc/ContentCompositorBridgeParent.cpp	Wed Jan 08 22:19:16 2020 +0000
+++ b/gfx/layers/ipc/ContentCompositorBridgeParent.cpp	Wed Jan 08 22:19:23 2020 +0000
@@ -18,7 +18,6 @@
 #  include "mozilla/gfx/DeviceManagerDx.h"  // for DeviceManagerDx
 #  include "mozilla/layers/ImageDataSerializer.h"
 #endif
-#include "mozilla/dom/WebGLCrossProcessCommandQueue.h"
 #include "mozilla/dom/WebGLParent.h"
 #include "mozilla/ipc/Transport.h"           // for Transport
 #include "mozilla/layers/AnimationHelper.h"  // for CompositorAnimationStorage
@@ -715,7 +714,7 @@
     return false;
   }
 #if defined(XP_WIN)
-  DeviceManagerDx* dm = DeviceManagerDx::Get();
+  gfx::DeviceManagerDx* dm = gfx::DeviceManagerDx::Get();
   MOZ_ASSERT(dm);
   if (!dm || !dm->GetCompositorDevice() || !dm->TextureSharingWorks()) {
     return false;
@@ -740,7 +739,7 @@
     return IPC_FAIL_NO_REASON(this);
   }
 
-  RefPtr<ID3D11Device> device = DeviceManagerDx::Get()->GetCompositorDevice();
+  RefPtr<ID3D11Device> device = gfx::DeviceManagerDx::Get()->GetCompositorDevice();
   if (!device) {
     return IPC_FAIL_NO_REASON(this);
   }